packages/mobile/e2e/Transfer.spec.js

```diff
@@ -32,48 +63,96 @@ describe('Transfer Works', () => {
   })
 
   it('NUX->Invite', async () => {
-    await expect(element(by.id('InviteWallTitle'))).toBeVisible()
-    await waitFor(element(by.id('ImportExistingUsingBackupKey')))
+    await waitFor(element(by.id('InviteCodeTitle')))
       .toBeVisible()
       .withTimeout(2000)
+
     await element(by.id('ImportExistingUsingBackupKey')).tap()
-    await waitFor(element(by.id('ImportWalletBakcupKeyInputField')))
+
+    await waitFor(element(by.id('ImportWalletBackupKeyInputField')))
       .toBeVisible()
       .withTimeout(2000)
-    await element(by.id('ImportWalletBakcupKeyInputField')).replaceText(SAMPLE_BACKUP_KEY)
-    await expect(element(by.id('ImportWalletButton'))).toBeVisible()
-    await element(by.id('ImportWalletButton')).tap()
-    await waitFor(element(by.id('ImportWalletButton')))
-      .toBeNotVisible()
-      .withTimeout(20000)
-    await waitFor(element(by.id('ImportWalletLoadingCircle')))
+
+    await sleep(1000)
```

Comment:
Can you clarify why we need these sleeps? Are they generally quite reliable or we still see flakiness with these tests?

Reply:
added a couple of comments